enhancements/update/update-status-api.md

```diff
@@ -0,0 +1,410 @@
+
+#### Standalone Clusters
+
+Standalone clusters are the primary target of the functionality and no special considerations are needed.
+
+#### Single-node Deployments or MicroShift
+
+USC and `UpdateStatus` API can work on SNO. USC is updated early in the procedure and will be able to expose the update status when the apiserver is available. USC and `UpdateStatus` can be disabled at installation time through the Capability mechanism for resource-sensitive SNO deployments. Existing SNO clusters using the `vCurrent` capability set would enable USC unless the cluster administrator intervenes by tweaking the capabilities before they update, which may warrant a release note when the feature goes GA.
+
+MicroShift is updated through `rpm-ostree` or `dnf`, so `UpdateStatus` API or any part of USC is not applicable and should not be included in MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+Full API proposal: https://github.com/openshift/api/pull/2012
+
+#### `UpdateStatus` API Overview
+
+```yaml
+apiVersion: update.openshift.io/v1alpha1
+kind: UpdateStatus
+metadata:
+  name: cluster
+spec: { }
+status:
+  controlPlane:
+    _: ...
+    informers:
+    - name: cvo-example-informer
+      insights: <list of insights reported by the informer>
+    - name: mco-example-informer
+      insights:
+      - type: ClusterVersion # CV status insight
+        _: ...
+      - type: ClusterOperator # CO status insight
+        _: ...
+      - type: UpdateHealth # General update health insight
+        _: ...
+    conditions: <list of standard kubernetes conditions>
+  workerPools:
+  - name: workers
+    _: ...
+    informers: <list of informers with reported insights>
+    conditions: <list of standard kubernetes conditions>
+  - name: infra
+    _: ...
+  conditions: <list of standard kubernetes conditions>    
+```
+
+The API has three conceptual layers:
+
+1. Through the innermost layer `.status.{controlPlane,workerPools[]}.informers`, the API exposes detailed information about individual concerns related to the update, called "Update Insights." The API is prepared to allow multiple external informers to contribute insights, but in this enhancement, the only informer is the USC itself.
+1. The aggregation layer `.status.{controlPlane,workerPools[]}` reports higher-level information about the update through this layer, serving as the USC's interpretation of all insights.
+1. The outermost layer, `.status.conditions`, is used to report operational matters related to the USC itself (the health of the controller and gathering the insights, not the health of the update process).
+
+We do not expect users to interact with `UpdateStatus` resources directly; the API is intended to be used mainly by tooling. A typical `UpdateStatus` instance is likely to be quite verbose.
```

Comment:
can we disambiguate "users" referring to specific Persona? e.g. Cluster Admin Persona